util/docs/macros/index.mjs

```diff
@@ -1017,12 +963,12 @@ function generateJavaScriptExampleResult(example) {
 }
 
 function generateRPCExample(example, m, moduleJson = {}) {
-    if (m.tags && m.tags.filter(t => (t.name === 'property-subscribe')).length) {
-        return generatePropertyChangedRPCExample(example, m, moduleJson)
-    }
-    else if (m.tags && m.tags.filter(t => (t.name === 'property-set')).length) {
-        return generatePropertySetRPCExample(example, m, moduleJson.info.title)
-    }
+    // if (m.tags && m.tags.filter(t => (t.name === 'property-subscribe')).length) {
```

Comment:
How come these are commented out?

Reply:
These examples are generated using the standard code now, removed the commented out code (and unused methods the comments referred to)